infoblox_client/objects.py

```diff
@@ -352,18 +352,27 @@ def __repr__(self):
                    for name in self._ea_dict)
         return "EAs:{0}".format(','.join(eas))
 
+    @staticmethod
+    def _value_to_bool(value):
```

Comment:
In oslo.config has pretty wide definition for bools, much wider than needed for EAs:

    TRUE_VALUES = ['true', '1', 'on', 'yes']
    FALSE_VALUES = ['false', '0', 'off', 'no']

My guess that for EAs we need to convert only 'True' and 'False' strings into bools (if need to convert any).
This idea came from discussions in here:
https://review.openstack.org/#/c/248345/1/networking_infoblox/tests/unit/common/test_ea_manager.py
https://review.openstack.org/#/c/248345/1/networking_infoblox/neutron/common/ea_manager.py